### ISSUE TITLE
Problem when creating a new task in a list other than Inbox

### ISSUE THREAD

**Issue by Rosalie-M** [idpaterson/alfred-wunderlist-workflow#73]
Hi,

Great workflow, superb interface and a lot of options. However, when using it, I noticed that there is an issue with saving a new task when I select "New task" and choose a list other than Inbox. I get this notification in the corner of my screen and my new task is nowhere to be seen:
![wunderlistbug](https://cloud.githubusercontent.com/assets/12778291/8021145/c7610866-0c61-11e5-9366-b76b44523d8b.png)
I'm using OS X Yosemite 10.10.3 and Wunderlist 3.2.2.

Thank you,

Rosalie

**Comment by idpaterson** [comment]
Interesting, so for some reason the command is outputting XML rather than the notification text. There are a few ways to figure out what XML is being generated here and why, but let's try this first since it might be the easiest. Open up the Terminal then paste the following command, replacing *List Name:Task* with the exact text you saw in Alfred before adding the task.
```bash
task="List Name:Task" && \
cd ~/Library/Application\ Support/Alfred\ 2/Alfred.alfredpreferences/workflows/*/wunderlist/.. && \
python alfred-wunderlist-workflow.py " $task" --commit && \
cd -
```

Thanks for reporting, hopefully this will show what's up!

**Comment by Rosalie-M** [comment]
It says that there is no such file or directory:
```
-bash: cd: /Users/rosaliemartel/Library/Application Support/Alfred 2/Alfred.alfredpreferences/workflows/*/wunderlist/..: No such file or directory
```
However, I just realized that there is an accentuated character in my List Name (I'm francophone). That seems to be the cause of the glitch, since I tried with other lists and I don't encounter this problem when creating a new task in a list without accentuated characters.

**Comment by idpaterson** [comment]
Ah thank you, I do have a test for that (*Jardinería: a sample task*) but it is not an end-to-end task so this must be failing somewhere else in the pipeline. I experienced the same XML issue when I added a list with an accent so that's all I will need from you for now. 

I'll update this issue once I've released the fix!

**Comment by idpaterson** [comment]
After quite a bit of testing I found two problems. First, the list name was breaking Alfred's notification within the notification script itself (a built-in Alfred component). I found that I need to encode the text being sent to that notification rather than just printing it as-is.

Second, if the text typed in Alfred has any accented characters, they were not being read correctly from the file that holds the Alfred text as of your very latest keystroke. 

So in my tests there were two separate conditions:
> Jardinería: a sample task
> Jard: a sámple task

Any accents typed into Alfred failed after reading the `.query` file before adding the task to Wunderlist.

> Jardineria: a sample task
> Jard: a sample task

No accents in the Alfred text but being added to the *Jardinería* list with accent, failed upon posting the notification, after the task was added to Wunderlist in the correct list.

Please let me know if [this updated build](https://copy.com/BDkX7uT6IPjW9C9K/Wunderlist.alfredworkflow?download=1) fixes this issue; if so I'll get 0.4.1 released right away!

**Comment by Rosalie-M** [comment]
This build seems to fix the issue. My list name had both a space and an accentuated character in it; the task name had spaces but no accentuated characters. 

Thank you!
